### PR TITLE
Adds support for props defined with array of strings

### PR DESCRIPTION
**Pull request by HelloDeadline**
PR for feature request https://github.com/vuedoc/parser/issues/23

### CODE REVIEW

**Comment by demsking** [comment]
Thank you for your contribution.

**Comment by demsking** [comment]
Could you add a test to cover the line [128 of the parser.js file please](https://coveralls.io/builds/17191647/source?filename=lib%2Fparser.js#L128)?

**Comment by demsking** [comment]
Could you also create your PR to the `develop` branch please?

**Comment by HelloDeadline** [comment]
@demsking Sure, I'll add test and change target branch to `develop` as soon as I have some free time (hopefully today). Thanks for quick response!

File: test/integration.spec.js [hunk 1]
```diff
@@ -181,6 +187,31 @@ function testComponentProps (optionsToParse) {
...
 
+describe('component.props (es6 Array)', () => {
+  let component = {}
+
+  parser.parse(optionsForPropsArray)
+    .then((_component) => (component = _component))
+    .catch((err) => { throw err })
+
+  it('should list props from string array', () => {
+    const propsNames = component.props.map((item) => item.name)
```
Comment (demsking) [inline test/integration.spec.js]: What about item.type? I think that `item.type` should be equal to `Any` in this case. What do you think?

**Comment by HelloDeadline** [comment]
You are right. 
But I think props types should be documented in a similar way for each notation. 
I'll open new PR next week with fixies for that one, if you okay with it.

**Pull request closed without merging**